src/EA.Weee.Web/Areas/Producer/Controllers/ProducerSubmissionController.cs

```diff
@@ -108,7 +108,10 @@ public async Task<ActionResult> ServiceOfNotice(bool? sameAsOrganisationAddress)
         {
             var model = mapper.Map<SmallProducerSubmissionData, ServiceOfNoticeViewModel>(SmallProducerSubmissionData);
 
-            model.Address = new ServiceOfNoticeAddressData();
+            if (model.Address == null)
+            {
```

Comment:
@will-hope98  If this change resolved a bug, really we should create a unit test to cover it. To keep the action as clean as possible I would move this into the mapper.

Reply:
Done